docs/book/src/topics/external-cloud-provider.md

```diff
@@ -23,3 +23,73 @@ kube-system   cloud-node-manager-hbbqt
 kube-system   cloud-node-manager-mfsdg                                            1/1     Running   0          39s
 kube-system   cloud-node-manager-qrz74                                            1/1     Running   0          24s
 ```
+
+## Storage Drivers
+
+### Azurefile CSI Driver
+
+Repository for Azurefile CSI Driver: https://github.com/kubernetes-sigs/azurefile-csi-driver
+
+After control plane is up and running, deploy the CSI Driver:
+
+```bash
+kubectl --kubeconfig=./${CLUSTER_NAME}.kubeconfig \
```

Comment:
If this is correct, I'm not sure if we want to add the manifests on our side.
I prefer to point to the repo and refer the install docs from those repos.

Reply:
just use below two links:
https://github.com/kubernetes-sigs/azuredisk-csi-driver/blob/master/docs/install-azuredisk-csi-driver.md
https://github.com/kubernetes-sigs/azurefile-csi-driver/blob/master/docs/install-azurefile-csi-driver.md